components/admin_console/admin_definition.jsx

```diff
@@ -2502,25 +2510,31 @@ export default {
                                     display_name: t('atmos/camo'),
                                     display_name_default: 'atmos/camo',
                                 },
+                                {
+                                    value: 'local',
+                                    display_name: t('local'),
+                                    display_name_default: 'local',
+                                },
                             ],
+                            isDisabled: needsUtils.stateValueFalse('ImageProxySettings.Enable'),
                         },
                         {
                             type: Constants.SettingsTypes.TYPE_TEXT,
-                            key: 'ServiceSettings.ImageProxyURL',
+                            key: 'ImageProxySettings.RemoteImageProxyURL',
                             label: t('admin.image.proxyURL'),
                             label_default: 'Image Proxy URL:',
```

Comment:
Maybe change the labels for both the URL and Options to "Remote Image Proxy URL" and "Remote Image Proxy Options" respectively? They aren't used by the local proxy, so it might be good to clarify that.